uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1000.php

```diff
@@ -36,11 +36,20 @@ private function up1000_updateHalfDayPmLabel() {
    * @param string $label
    */
   private function up1000_updateOptionValueLabel($name, $label) {
-    civicrm_api3('OptionValue', 'get', [
-      'sequential' => 1,
-      'option_group_id' => 'hrleaveandabsences_leave_request_day_type',
-      'name' => ['IN' => [$name]],
-      'api.OptionValue.create' => ['id' => '$value.id', 'label' => $label],
-    ]);
+    try {
+      civicrm_api3('OptionValue', 'get', [
```

Comment:
@mickadoo the short answer is: I don't know. Checking the order of the method calls was the first thing I did, and I agree that the option groups should be available at this point, but for some reason that I don't know why, they don't. I was able to reproduce this in 2 different sites. I didn't think it was worth to investigate further to found out what was happening.

The real problem here is that running the upgraders during the installation is something "wrong". We only do this to avoid duplicating installation code on the upgraders.